models/demos/t3000/falcon40b/tests/ci/test_falcon_end_to_end_60_layer_t3000_prefill_10_loops.py

```diff
@@ -98,18 +98,18 @@ def test_FalconCausalLM_prefill_end_to_end_t3000_ci_loops_10(
     if data_type == "BFLOAT8_B":
         if seq_len == 32:
             out_pcc = 0.983
-            k_cache_pcc = 0.985
-            v_cache_pcc = 0.957
+            k_cache_pcc = 0.982
+            v_cache_pcc = 0.949
             token_pcc = 0.99
         elif seq_len == 128:
             out_pcc = 0.990
-            k_cache_pcc = 0.990
-            v_cache_pcc = 0.963
+            k_cache_pcc = 0.989
+            v_cache_pcc = 0.950
             token_pcc = 0.99
         elif seq_len == 2048:
             out_pcc = 0.993
-            k_cache_pcc = 0.992
-            v_cache_pcc = 0.979
+            k_cache_pcc = 0.991
+            v_cache_pcc = 0.972
```

Comment:
Are these minimal PCC for all layers? Are the 60th layer PCCs 0.99 still?

Reply:
Yes, that's the minimum over all layers. I'm currently not printing the last one, number 56 would be 0.99 (kcache) and 0.98 (vcache) for S=2k for example

Reply:
I guess I'd just make sure we still keep the final-layer PCC good - not printing it is just a consequence of reducing the noise I think, but we should be aware about the last one imo

Reply:
Why do you think the last one is more important then the earlier layers' kvcaches? I think the range / the minimum over all layers is something we should look at. The test is checking all layers against that threshold, so all layers have to be higher than that one.

Reply:
I'd say because the last layer PCC looks at the values that later on go to the decode phase, however no strong opinion here that the others are 'less' important

`The test is checking all layers against that threshold`
Seems to me that only each 4th layer is checked [from here](https://github.com/tenstorrent/tt-metal/blob/main/models/demos/t3000/falcon40b/tests/test_falcon_end_to_end.py#L317), but I might be missing something

Reply:
Yeah you're right. I thought we only print every 4th layer 🤔 maybe I'll add that check back in for all layers then.

Regarding kvcaches: they all go to decode and earlier layers are going to have larger impact on the decode model because the earlier an error occurs (based on earlier kv caches) in the model the larger the perturbation in the end I think. So at least to me it's definitely not clear that later layers would have larger impact - if anything less imo - but also just an intuition and nothing proven out of course. In any case, I think we should look at all of them and not only the last one.

Reply:
I see, makes sense then!

I agree that the check should be added for all layers.